react-common/components/controls/Button.tsx

```diff
@@ -21,6 +21,7 @@ export interface ButtonViewProps extends ContainerProps {
     ariaPosInSet?: number;
     ariaSetSize?: number;
     ariaSelected?: boolean;
+    ariaChecked?: boolean | "mixed";
```

Comment:
Hm, I thought mixed was more for those scenarios where you have a parent check box and some of the children are checked (it often gets filled in instead of checked). Should we default to undefined instead? Or are we just relying on the fact that aria-checked isn't normally supported on the button role anyway?

Reply:
Thanks for calling this out. I'd misremembered. It was `aria-pressed` I wanted to expose: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#aria_state_information

Reply:
Fixed